peer/brontide.go

```diff
@@ -2408,6 +2426,15 @@ func (p *Brontide) reenableActiveChannels() {
 func (p *Brontide) fetchActiveChanCloser(chanID lnwire.ChannelID) (
 	*chancloser.ChanCloser, error) {
 
+	chanCloser, found := p.activeChanCloses[chanID]
```

Comment:
Why's it necessary to move around this check?

Reply:
On restart if coop-close has already started, we won't load the link or add an entry in `activeChannels`

Reply:
This might have to change later when the link needs to be added on restart and we need to get to a clean state.

Reply:
A consequence of this check is having to delete from the map when finalizeChanClosure is called - otherwise the itest would fail (cosmetically). Plus it makes sense to delete it from the map once things are wrapped up

Reply:
not a big deal but this should not really be in this refactor commit since it is a slight behaviour change. Also should be in the same commit as the `delete from the map when finalizeChanClosure is called` change